tox.ini

```
@@ -184,6 +184,9 @@ envlist =
     {py3.9,py3.11,py3.12}-openai-latest
     {py3.9,py3.11,py3.12}-openai-notiktoken
 
+    # OpenFeature
+    {py3.8,py3.12,py3.13}-openfeature-latest
```

```suggestion
    {py3.8,py3.12,py3.13}-openfeature-v0.7
    {py3.8,py3.12,py3.13}-openfeature-latest
```

See my comment below for rationale